samples/app/hello-world/src/main/AndroidManifest.xml

```diff
@@ -10,7 +10,9 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
 
-        <activity android:name="com.badoo.ribs.samples.helloworld.app.RootActivity">
+        <activity
+            android:name="com.badoo.ribs.samples.helloworld.app.RootActivity"
+            android:exported="true">
```

Comment:
`false`

Reply:
> RibsViewActivity

This applies only to Launcher activities, and they are not. So it should compile just fine